PixelFactory/src/PixelFactory.cpp

```diff
@@ -72,5 +72,5 @@ int main()
 	pf.run();
 
 	cout << "shutting down" << endl;
-	return 0;
+	return (int)0;
```

Comment:
still segfaults instead of exiting nicely. Probably a slow joiner problem, but that's on Widget. Do I need a Widget::Shutdown()?